### ISSUE TITLE
Settings Editor: cannot use keyboard to navigate between user/workspace/folder settings

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#39697]
Testing: https://github.com/Microsoft/vscode/issues/35580

I am not able to reach the other settings categories using the keyboard.
